test/s3select_test.cpp

```diff
@@ -3392,3 +3392,103 @@ TEST(TestS3selectFunctions, test_trim_expressions)
 
   ASSERT_EQ(s3select_result_3, s3select_result_4);
 }
+
+TEST(TestS3selectFunctions, truefalse)
```

Comment:
please review /test_single_column_single_row( ) /
it saves some code-lines. 

generally should modify other tests,also.